nameparser/config/titles.py

```diff
@@ -37,12 +37,15 @@
 TITLES = FIRST_NAME_TITLES | set([
     "attaché",
     "chargé d'affaires",
+    "chargé d’affaires",
     "king's",
+    "king’s",
```

Comment:
Suggestion from my side would be to normalize the apostrophes and dashes prior to matching against these lists. There are too many variations to capture them all.

Reply:
Absolutely agreed; I did not take the time to take a look if this happens already somewhere in the code… Obviously there's a bunch of misused characters that might function as apostrophes, even if we only take into account latin script, e.g. `'’‘'` and the backtick that \` creates the `<code/>`. Search the unicode.org charts for all other apostrophes for other writing systems

Reply:
I would benefit from this MR as I will have to deal with German names soon. Would it be reasonable to move the debate about apostrophes to a dedicated issue so that the rest of the contribution could be released?

Reply:
Fine by me. The PR is quite old. So I guess it needs to have conflicts resolved etc.